tests/acs/test_unit.py

```diff
@@ -512,11 +504,11 @@ def test_blot_with_lan5(self):
         assert(med_diff < 1.0e-6)
         assert(max_diff < 1.0e-5)
 
-    def test_blot_with_image(self):
+    def not_test_blot_with_image(self):
```

Comment:
I counted 14 methods in this file but pytest is only running 11 of them, which means it is not collecting those named "not_test*". Is this intentional?

Reply:
That is intentional for the time being.  Those which have been renamed 'not_test' are being included but 'skipped' due to longer term issues with those specific test definitions. These are intended to be updated and turned on in a future PR.

Reply:
Why not mark them as `xfail`? It would be more obvious that way, no?

Reply:
I will evaluate what would be best for these tests, looking to mark them as 'xfail' where possible.

Reply:
1 test was reset to use 'xfail', with possible reason stated, while the other 2 cases you identified were revised so that they no longer needed 'xfail', but would pass.